packages/terra-divider/src/Divider.jsx

```diff
@@ -29,11 +45,14 @@ const Divider = (props) => {
   );
 
   if (!text) {
-    return <hr {...customProps} className={dividerClassNames} aria-hidden="true" />;
+    return <hr {...customProps} className={dividerClassNames} />;
+  }
+  if (level) {
```

Comment:
should we add a check here to check if the text prop value is also being passed ?

Reply:
the text is checked the above line no 47. if there is no text it won't  come to this line.it will come to the line no 50 when text has some value.